From df62e4cb4b4e1ca6530458083bf009cdede7538f Mon Sep 17 00:00:00 2001
From: Nick Garvey <garvey.nick@gmail.com>
Date: Fri, 23 Jun 2017 21:02:44 -0700
Subject: [PATCH] Make npm hook scripts executable

This is required for npm install to work properly
Without this patch, this is the error

https://gist.github.com/anonymous/a45fd3c41021f8849d9272cf59108c2b#file-npm-debug-log-L232-L233

With this patch, the command ran successfully
---
 init_dist/node_modules/.hooks/install       | 0
 init_dist/node_modules/.hooks/install.cmd   | 0
 init_dist/node_modules/.hooks/uninstall     | 0
 init_dist/node_modules/.hooks/uninstall.cmd | 0
 4 files changed, 0 insertions(+), 0 deletions(-)
 mode change 100644 => 100755 init_dist/node_modules/.hooks/install
 mode change 100644 => 100755 init_dist/node_modules/.hooks/install.cmd
 mode change 100644 => 100755 init_dist/node_modules/.hooks/uninstall
 mode change 100644 => 100755 init_dist/node_modules/.hooks/uninstall.cmd

diff --git a/init_dist/node_modules/.hooks/install b/init_dist/node_modules/.hooks/install
old mode 100644
new mode 100755
diff --git a/init_dist/node_modules/.hooks/install.cmd b/init_dist/node_modules/.hooks/install.cmd
old mode 100644
new mode 100755
diff --git a/init_dist/node_modules/.hooks/uninstall b/init_dist/node_modules/.hooks/uninstall
old mode 100644
new mode 100755
diff --git a/init_dist/node_modules/.hooks/uninstall.cmd b/init_dist/node_modules/.hooks/uninstall.cmd
old mode 100644
new mode 100755
-- 
2.7.4

